myst_nb/static/mystnb.css

```diff
@@ -217,6 +217,73 @@ tbody span.pasted-inline img {
   max-height: none;
 }
 
+
+/* Adding scroll bars if tags: output_scroll, scroll-output, and scroll-input
+ * On screens, we want to scroll, but on print show all
+ *
+ * It was before in https://github.com/executablebooks/sphinx-book-theme/blob/eb1b6baf098b27605e8f2b7b2979b17ebf1b9540/src/sphinx_book_theme/assets/styles/extensions/_myst-nb.scss
+*/
+
+div.cell.tag_output_scroll div.cell_output, div.cell.tag_scroll-output div.cell_output {
+	 max-height: 24em;
+	 overflow-y: auto;
+	 max-width: 100%;
+	 overflow-x: auto;
+}
+
+div.cell.tag_output_scroll div.cell_output::-webkit-scrollbar, div.cell.tag_scroll-output div.cell_output::-webkit-scrollbar {
+	 width: 0.3rem;
+	 height: 0.3rem;
+}
+
+div.cell.tag_output_scroll div.cell_output::-webkit-scrollbar-thumb, div.cell.tag_scroll-output div.cell_output::-webkit-scrollbar-thumb {
+	 background: #c1c1c1;
```

Comment:
these colors should be added as CSS variables, e.g. `var(--mystnb-source-border-color)`
have a look at the the top of this file for how they are specified.

also see https://github.com/executablebooks/MyST-NB/issues/443, for why we want to do this; it makes it easier for others to adapt the CSS for integration with their themes

Reply:
Thanks. That's good practice.